core/trino-spi/src/main/java/io/trino/spi/predicate/TupleDomain.java

```diff
@@ -422,18 +422,14 @@ public String toString()
 
     public String toString(ConnectorSession session)
     {
-        StringBuilder buffer = new StringBuilder();
         if (isAll()) {
-            buffer.append("ALL");
-        }
-        else if (isNone()) {
-            buffer.append("NONE");
+            return "TupleDomain{ALL}";
```

Comment:
Consitency is good. But wouln't it be better to drop the `TupleDomain` prefix. Should be obvious from the context.

Reply:
I can do that too